src/lib/journey/callbacks/string-attribute/string-attribute-input.svelte

```diff
@@ -1,26 +1,31 @@
 <script lang="ts">
   import {
-    getAttributeValidationFailureText,
     getInputTypeFromPolicies,
     isInputRequired,
-  } from '$journey/utilities/callback.utilities';
+  } from '$journey/callbacks/_utilities/callback.utilities';
   import type { AttributeInputCallback } from '@forgerock/javascript-sdk';
 
+  import {
+    getValidationFailures,
+  } from '$journey/callbacks/_utilities/callback.utilities';
   import Input from '$components/compositions/input-floating/floating-label.svelte';
-  import { interpolate } from '$lib/utilities/i18n.utilities';
+  import { interpolate } from '$lib/_utilities/i18n.utilities';
+  import Policies from '$journey/callbacks/_utilities/policies.svelte';
```

Comment:
Similar changes to this and the `ValidateCreateUsername` callback handler since they all have policies.